src/include/miopen/env.hpp

```diff
@@ -207,6 +208,13 @@ void UpdateEnvVar(EnvVar, const ValueType& val)
     EnvVar::Ref().UpdateValue(val);
 }
 
+template <typename EnvVar>
+void UpdateEnvVar(EnvVar, const std::string_view& val)
```

Comment:
Minor suggestion: rename to ParseEnvVar or something like that and add a comment saying that this is for tests only and normally folks should use `UpdateEnvVar` (the old one).

Reply:
I think none of them should be used normally. All that functionality has been introduced exceptionally for the testing purposes.
Let's rename it a bit later, after we finished the most urgent gtests tickets (just to avoid unnecessary changes for everyone).

Reply:
[R] @CAHEK7 @amberhassaan 
> All that functionality has been introduced exceptionally for the testing purposes.

We can enclose this into `miopen::debug` namespace to emphasize this fact.